client/session_test.go

```diff
@@ -0,0 +1,86 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
+	"github.com/nspcc-dev/neofs-api-go/v2/session"
+	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
+	"github.com/nspcc-dev/neofs-sdk-go/crypto/test"
+	"github.com/stretchr/testify/require"
+)
+
+type emptySessionID struct {
+	signer neofscrypto.Signer
+}
+
+func (m emptySessionID) CreateSession(_ *client.Client, _ *session.CreateRequest, _ ...client.CallOption) (*session.CreateResponse, error) {
+	var body session.CreateResponseBody
+	body.SetSessionKey([]byte{1})
+
+	var resp session.CreateResponse
+	resp.SetBody(&body)
+	if err := signServiceMessage(m.signer, &resp); err != nil {
+		panic(fmt.Sprintf("signServiceMessage: %v", err))
+	}
+
+	return &resp, nil
+}
+
+type emptySessionKey struct {
+	signer neofscrypto.Signer
+}
+
+func (m emptySessionKey) CreateSession(_ *client.Client, _ *session.CreateRequest, _ ...client.CallOption) (*session.CreateResponse, error) {
+	var body session.CreateResponseBody
+	body.SetID([]byte{1})
```

Comment:
how about making a single implementation with parameterized ID and key?

Reply:
removed second implementation